internals/secrethub/env_source.go

```diff
@@ -663,3 +669,26 @@ func (o *osEnv) env() (map[string]value, error) {
 	}
 	return res, nil
 }
+
+type MapValue struct {
+	stringMap map[string]string
+}
```

Comment:
We could instead use the [StringToString flag type](https://godoc.org/github.com/spf13/pflag#StringToString) from Cobra.

Reply:
Nice one. Adjusted to your idea and it is better.